integration/kubernetes/helm-chart/alluxio/CHANGELOG.md

```diff
@@ -159,3 +159,7 @@
 0.6.16
 
 - Change helm-chart fuse hostPath type from File to CharDevice
+
+0.6.17
+
+- Add hostAliases in Master and Worker Pods
```

Comment:
@apc999 Just double checking, FUSE only talks to the Alluxio masters and workers right? When Alluxio(in K8s) is using HDFS(outside K8s on physical nodes, NAT provided from `hostAliases`) as UFS, the FUSE pods do NOT talk to HDFS in any way right?

Reply:
The `hostAliases` are only added to the masters and workers. This means if the FUSE pods want to talk to the HDFS, they won't be able to resolve those IPs.

Reply:
@jiacheliu3  I've tested in a fuse pod without `hostAliases`. I've created a new folder in `/mnt/alluxio-fuse/` and create a new file to this folder. It looks like except for a root permission error at the beginning, it is normal to create folders and files and write contents.